addons/pkg/constants/constants.go

```diff
@@ -97,4 +97,10 @@ const (
 
 	// CRDWaitPollTimeout is poll timeout for checking server resources
 	CRDWaitPollTimeout = time.Minute * 10
+
+	// DataValueFileName is the name of data value file
+	DataValueFileName = "values.yaml"
+
+	// CalicoAddonName is the name of Calico addon
+	CalicoAddonName = "calico"
```

Comment:
Nit: I think this constant can be put under `calicoconfig_controller.go`. This is because we'll hand over the controller to other teams, let's separate the code as much as possible.